src/oc/web/components/fullscreen_post.cljs

```diff
@@ -193,6 +201,61 @@
         (reset! (::initial-headline s) initial-headline)
         (reset! (::edited-data-loaded s) true)))))
 
+(defn remove-video []
```

Comment:
I'm not really certain why there is duplicated here and https://github.com/open-company/open-company-web/pull/587/files#diff-c257f209ba0537d00a3b216f0e4d992aR150

I can't seem to find where this is used in full screen.